[2024-01-13T21:39:30.419+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2024-01-13T21:23:58.788584+00:00 [queued]>
[2024-01-13T21:39:30.427+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2024-01-13T21:23:58.788584+00:00 [queued]>
[2024-01-13T21:39:30.429+0000] {taskinstance.py:2171} INFO - Starting attempt 2 of 2
[2024-01-13T21:39:30.439+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2024-01-13 21:23:58.788584+00:00
[2024-01-13T21:39:30.446+0000] {standard_task_runner.py:60} INFO - Started process 1055 to run task
[2024-01-13T21:39:30.452+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'manual__2024-01-13T21:23:58.788584+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp7t97g1js']
[2024-01-13T21:39:30.456+0000] {standard_task_runner.py:88} INFO - Job 14: Subtask local_to_gcs_task
[2024-01-13T21:39:30.476+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-01-13T21:39:30.521+0000] {task_command.py:423} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2024-01-13T21:23:58.788584+00:00 [running]> on host 6bca031b1185
[2024-01-13T21:39:30.581+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='local_to_gcs_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-13T21:23:58.788584+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-13T21:23:58.788584+00:00'
[2024-01-13T21:39:37.320+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-13T21:39:37.339+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20240113T212358, start_date=20240113T213930, end_date=20240113T213937
[2024-01-13T21:39:37.369+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-13T21:39:37.397+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
